### ISSUE TITLE
No way to type "\" or "|" (KC_BSLASH)  in Ginny keymaps?

### ISSUE THREAD

**Issue by russsharek** [qmk-combos/combos#4]
I was poking at en-keymap.def, num-keymap.def, and cmd-keymap.def, and there doesn't seem to be a mapping for the backslash/pipe key.

I did notice that there was a doubled mapping for KC_MINS on line 28 and 29:

`PRES(NUM|AE|AO,                                                                                   KC_MINS)
PRES(NUM|AS|AE,                                                                                   KC_MINS)`

I was wondering if one of these was in error, and perhaps it was intended to be mapped to the missing KC_BSLASH key?

**Comment by germ** [comment]
Yup that's definatley an oversight on my part. I'll try and get that fixed up later today

**Comment by FCO** [comment]
Any news about this one?
